scss/components/_card.scss

```diff
@@ -67,7 +67,7 @@ $card-margin: $global-margin !default;
   box-shadow: $shadow;
 
   overflow: hidden;
-  color: $card-font-color;
+  color: $color;
```

Comment:
So basically you would like that user can't change the color and it should always be `$card-font-color` ?? 
Am i correct in understanding you ?